lib/bing_ads_ruby_sdk/json_api.rb

```diff
@@ -0,0 +1,64 @@
+# frozen_string_literal: true
+
+require "bing_ads_ruby_sdk/header"
+require "bing_ads_ruby_sdk/oauth2/authorization_handler"
+require "bing_ads_ruby_sdk/services/json"
+
+module BingAdsRubySdk
+  class JsonApi
```

Comment:
I was wondering about making this a base class and creating a CampaignManagement, CustomerManagement 
 etc. services which inherit from it.

Reply:
probably a good idea

Reply:
I probably meant `lib/bing_ads_ruby_sdk/services/json.rb`